File: Codeeval/read_more.php
<?php

	$fh = fopen($argv[1],'r');
	

	while(! feof($fh) )
	{
		$testcase = trim(fgets($fh));
		if($testcase == "") continue;
		

		if(strlen($testcase) <= 55)
		{
			echo $testcase;
			echo PHP_EOL;
		}
		else
		{
			$out = substr($testcase,0,40);	
			
			if(strrpos($out,' ') != False)
			{
				$lastspc = strrpos($out,' ');
				$out = substr($out, 0,$lastspc);

			}

			echo $out."... <Read More>";
			echo PHP_EOL;
			
		}
		
	}


	function count_array_size($lines)
	{
		$cnt = 0;
		foreach($lines as $line)
		{
			$cnt += strlen($line);
		}
		return $cnt;
	}

File: Codeeval/read_more.py
cef = open("read_more.txt", "r")
for line in cef:	
	if len(line) <= 55:
		print(line)
	else:
		output = line[:40]
		indx = output.rfind(" ")	
		if indx != -1:
			output = output[:indx]
		output += "... <Read More>"	
		print(output)
	
	


